index.bs

```diff
@@ -4971,21 +4978,21 @@ partial interface MLGraphBuilder {
         - <dfn>input</dfn>: an {{MLOperand}}. The input tensor.
         - <dfn>options</dfn>: an optional {{MLReduceOptions}}. The optional parameters of the operation.
 
-    **Returns:** an {{MLOperand}}. The reduced output tensor.
+    **Returns:** an {{MLOperand}}. The reduced output tensor. If the input operand is a scalar, the reduction function is applied to the scalar value and the output is also a scalar.
 </div>
 
-<div class="note">
+<div>
     **Reduction types:**
-        - *L1*: Compute the <a href="https://mathworld.wolfram.com/L1-Norm.html">L1 norm</a> of all the input values along the axes.
-        - *L2*: Compute the <a href="https://mathworld.wolfram.com/L2-Norm.html">L2 norm</a> of all the input values along the axes.
-        - *LogSum*: Compute the log value of the sum of all the input values along the axes.
-        - *LogSumExp*: Compute the log value of the sum of the exponent of all the input values along the axes.
-        - *Max*: Compute the maximum value of all the input values along the axes.
-        - *Mean*: Compute the average value of all the input values along the axes.
-        - *Min*: Compute the minimum value of all the input values along the axes.
-        - *Product*: Compute the product of all the input values along the axes.
-        - *Sum*: Compute the sum of all the input values along the axes.
-        - *SumSquare*: Compute the sum of the square of all the input values along the axes.
+        - *L1*: Compute the <a href="https://mathworld.wolfram.com/L1-Norm.html">L1 norm</a>, the sum of the absolute value of all the input values.
+        - *L2*: Compute the <a href="https://mathworld.wolfram.com/L2-Norm.html">L2 norm</a>, the square root of the sum of the square of all the input values.
+        - *LogSum*: Compute the log value of the sum of all the input values.
+        - *LogSumExp*: Compute the log value of the sum of the exponent of all the input values.
+        - *Max*: Compute the maximum value of all the input values.
+        - *Mean*: Compute the average value of all the input values.
```

Comment:
Removing "along the axes" now makes this "Compute the average value of all the input values" sound like it finds the average of *all* values in the input tensor 🤨, but it only finds the average within each related *group* of shared input values (sharing the same axis, of those being reduced). Impetus to remove it?

Proposals:
- Remove "all":
```suggestion
        - *Mean*: Compute the average value of the input values.
```
- Add "related":
```suggestion
        - *Mean*: Compute the average value of all the related input values.
```
- Restore "along the axes":
```suggestion
        - *Mean*: Compute the average value of all the input values along the axes.
```

Reply:
How about "selected input values" ?

The impetus for removing "along the axes" is that if no axes are selected the operation still happens. Plus it's very repetitive.

Reply:
I like removing "all". Let's try that.

Reply:
I think more explanation of how reduction works across axes is good, but ISTM that keeping these definitions of the different functions concise is important. Anything more we want to add to text around the axes parameter?

Reply:
> I like removing "all". Let's try that. [f982418](https://github.com/webmachinelearning/webnn/commit/f9824188789c2c0653764c1538a0db799978179c)

Fine with me.

> Anything more we want to add to text around the axes parameter?

Maybe later gator, but happy to merge.